app/stylesheets/global.scss

```diff
@@ -0,0 +1,12 @@
+@import 'config/timing';
+@import 'config/type';
+
+html,
+body {
+  font-family: 'Open Sans', sans-serif;
+}
+
+a {
+  @include link-transition( 600ms, ease);
```

Comment:
I like the 2nd option since we won't need to have an extra scss file for timing. :)